frontend/providers/PHProvider.tsx

```
@@ -1,16 +1,60 @@
+
+    if (status === "authenticated" && session?.user?.email) {
+      (async () => {
+        try {
+          const authUser = await getAuthorizedUserByEmail(session.user.email!);
+          if (cancelled) return;
+
+          if (authUser?.id) {
+            ph.identify(authUser.id.toString(), {
+              name: session.user.name,
+              email: session.user.email,
+              username: (session.user as any).username,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)